### ISSUE TITLE
Entity Pre-set Files

### ISSUE THREAD

**Issue by CastAlmania** [bridge-core/editor#1100]
**Summary**
In the toolbar of Bridge V2, there should be an option in the 'help' section or just aligned with the other options of selecting an entity pre-set file, so you can edit the behaviour of mobs/animals in-game. This would help speed up add-on development that utilises entity files, and would also make finding entity templates a lot easier for users.

**Alternatives/Considerations**
Having a method of finding entity pre-sets in the help documents.

**Additional context/Motivation**
--
![image_2024-03-27_213021434](https://github.com/bridge-core/editor/assets/111056771/a7706af9-4a45-48f6-8d1a-414c392a5ffa)

**Comment by outercloudstudio** [comment]
Are you suggesting being able to view vanilla entity files?

**Comment by CastAlmania** [comment]
Pretty much, while also being able to just copy it into your add-on, it'd make things a lot easier.

**Comment by outercloudstudio** [comment]
I agree that this could be useful but I'm not sure how to implement it yet. I'll have to do more thinking.

**Comment by CastAlmania** [comment]
Alright!
